assets/audit-logs/policy.yaml

```diff
@@ -0,0 +1,107 @@
+apiVersion: audit.k8s.io/v1beta1
+kind: Policy
+rules:
+# non relevant rules
+- level: None
+  nonResourceURLs:
+  - "/healthz*"
+  - "/logs"
+  - "/metrics"
+  - "/swagger*"
+  - "/version"
+
+# extended audit of auth delegation
+- level: RequestResponse
+  omitStages:
+  - RequestReceived
+  resources:
+  - group: authorization.k8s.io
+    resources:
+    - subjectaccessreviews
+
+# NP - NG
```

Comment:
Should we drop this part? Not relevant to the workshop.

Reply:
👍 just copy/pasted ours will update